packages/core/admin/server/services/api-token.js

```diff
@@ -107,11 +107,21 @@ const revoke = async id => {
   return strapi.query('admin::api-token').delete({ select: SELECT_FIELDS, where: { id } });
 };
 
+/**
+ * @param {string|number} id
+ *
+ * @returns {Promise<Omit<ApiToken, 'accessKey'>>}
+ */
+const get = async id => {
```

Comment:
What do you think of renaming it `getById`?

Reply:
I tend to avoid getting to verbose when not necessary. IMHO as the `id` is the primary identifier for a record (api token) we can omit the `ById` as it feel redundant. But I do understand the reasoning.

Both options are valid. If we are suffixing the `get` methods with `ByX` I'll update mine to stay consistent.

Reply:
Yes we tend to have `ByX` if the function doesn't get where params. Thank you for the update!